### PR TITLE
message validation: check voluntary exit existence

### PR DESCRIPTION
**Pull request by nkryuchkov**
According to https://github.com/ssvlabs/ssv/pull/1393#discussion_r1667687976

### CODE REVIEW

File: operator/duties/dutystore/voluntary_exit.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import (
+	"sync"
+
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+)
+
+type VoluntaryExitDuties struct {
+	mu sync.RWMutex
+	m  map[phase0.Slot]map[phase0.BLSPubKey]int
```
Comment (y0sher) [inline operator/duties/dutystore/voluntary_exit.go:11]: we need to clean this map at some point, maybe remove the duties after they are executed? or that's too short?

**Review by y0sher — CHANGES_REQUESTED** [review]
clean the new duty store you added.

File: operator/duties/dutystore/voluntary_exit.go (hunk 1 above)
Comment (nkryuchkov) [reply operator/duties/dutystore/voluntary_exit.go:11]: done, let's clean them after 1 epoch

**Comment by moshe-blox** [comment]
@nkryuchkov @y0sher 
another refactor is required for this to work, since the event handler currently doesnt fire tasks if its not your validator:
https://github.com/ssvlabs/ssv/blob/adaa86b1da7a742c27467622427e089adb8a1148/eth/eventhandler/handlers.go#L514-L516

we should probably try something like this:
image

and in DutyScheduler always add to duties map but only execute our own duties

**Review by y0sher — APPROVED** [review]
lgtm.

**Pull request merged**